checkstyle-tester/README.md

```diff
@@ -224,6 +224,25 @@ Sevntu's current version must be referenced at https://github.com/checkstyle/con
 
 Finally, just run checkstyle-tester as described above.
 
+## ANTLR Grammar Change Testing Procedure
+
+After modifying Checkstyle's ANTLR grammar, it is neccessary to compare how Checkstyle's checks are affected by your changes.  This is done by comparing the check violation output of your modified branch of Checkstyle against Checkstyle's master branch, and comparing the abstract syntax trees of both branches. 
+
+#### Check Regression Report
+
+To generate the check difference report, you must separate the modules and settings in the [`checkstyle_checks.xml`](https://github.com/checkstyle/checkstyle/blob/master/config/checkstyle_checks.xml) configuration file, found in the checkstyle repository, into separate configuration files (this helps avoid OOM).  The best way to separate the modules is to determine which typically produce the most violations, such as Indentation and WhiteSpace Checks, and put them each into their own configuration file, then separating the rest into files with roughly ten modules each. You may modify all the modules that depend on external files to use default settings.  **For each of the configuration files, you should use the [`my_check.xml`](https://github.com/checkstyle/contribution/blob/master/checkstyle-tester/my_check.xml) file as a base**, and add the modules from `checkstyle_checks.xml` to it. Then `diff.groovy` should be run on all projects in `projects-to-test-on.properties`, using the `diff.groovy` script once for each configuration file.  [See instructions above](https://github.com/checkstyle/contribution/tree/master/checkstyle-tester#basic-difference-report) for "Basic Difference Report".
```

Comment:
> this helps avoid OOM
> Note: if you are experienceing maven OOM, see https://cwiki.apache.org/confluence/display/MAVEN/OutOfMemoryError

I feel it would be beneficial to specify what OOM is as many may not know it by that short name.

Reply:
Fixed.